Pipfile

```diff
@@ -14,6 +14,7 @@ tqdm = "*"
 imageio = {extras = ["ffmpeg"], version = "*"}
 torchvision = "*"
 click = "*"
+pylint = "*"
```

Comment:
should definitely be in dev and not in regular packages.
It's a big no no usually since it's a GPL license as well